ahnlich/types/src/ai/mod.rs

```diff
@@ -12,8 +12,14 @@ use crate::keyval::StoreInput;
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum AIModel {
     // Image model
-    DALLE3,
+    Dalle3,
```

Comment:
Would leave a TODO here to get rid of those fake `Llama3` and `Dalle3` entries that we noobs added afterwards... now that we have actual models

Reply:
I intend removing it when the PR is done.